tests/features/api/api_intake.feature

```diff
@@ -96,22 +96,22 @@ Feature: Intake API
         # And ...
 
         Examples:
-            | user        | collection                      |
-            | datamanager | /tempZone/yoda/home/grp-initial |
-            | researcher  | /tempZone/yoda/home/grp-initial |
+            | user        | collection                        |
+            | datamanager | /tempZone/home/grp-intake-initial |
+            | researcher  | /tempZone/home/grp-intake-initial |
 
 
     Scenario Outline: Get all details for a dataset
         Given user <user> is authenticated
         And dataset exists
```

Comment:
This step (dataset exists) can probably be removed, because it does not do anything since the dataset_id is specified in the examples?